blockbase/block-templates/singular.html

```diff
@@ -8,9 +8,9 @@
 
 <!-- wp:group {"tagName":"main","layout":{"inherit":true}} -->
 <main class="wp-block-group post-content">
-<!-- wp:post-featured-image {"align":"full"} /-->
+<!-- wp:post-featured-image {"align":"wide"} /-->
```

Comment:
We probably want to change this back, but it's useful for demonstrating the problem